frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -66,7 +71,13 @@ export const StaticEmbeddingOptionCard = () => {
 
   return (
     <EmbeddingOption
-      icon={enabled ? <StaticEmbeddingOn /> : <StaticEmbeddingOff />}
+      icon={
+        <StaticEmbedding
```

Comment:
Fix the icon not following custom brand color. Another benefit is we only need to maintain one icon file.